server/src/unifyfs_request_manager.c

```diff
@@ -84,6 +86,8 @@ do { \
  * to process the read replies. It iterates with the client until
  * all incoming read replies have been transferred. */
 
+typedef struct extents_list extents_list;
```

Comment:
this typedef should be in the same place as the struct definition

Reply:
fixed